### PR TITLE
Improves configuration docs for Admin

### LINKED ISSUE #57: Setup instruction don't quite match Bitbucket UI
The instructions tell me to setup oAuth under _Access Management_ however, this should say under _Apps and Features_ which matches the current Bitbucket admin UI. 
image

Current Admin UI:
image

### PR DESCRIPTION
**Pull request by sanjaydemansol**
Now,

Shot_08012022_n4VMzsnV@2x

#### Ticket Link

Fixes #57

### CODE REVIEW

File: docs/admin-guide/configuration.md [hunk 1]
```diff
@@ -5,22 +5,23 @@ Configuration is started in Bitbucket and completed in Mattermost.
 ## Step 1: Register an OAuth Application in Bitbucket
 
 1. Go to [https://bitbucket.org](https://bitbucket.org) and log in.
-2. Go the **Settings** page for your organization.
-3. Go to **Access Management &gt; OAuth**.
-4. Select **Add consumer** and set the following values:
+2. From your profile avatar in the bottom left, click on the workspace in the **Recent workspaces** list or click **All workspaces** to open an entire list from which to choose.
```
Comment (justinegeffen) [inline docs/admin-guide/configuration.md]:
```suggestion
2. From your profile avatar in the bottom left, select the workspace in the **Recent workspaces** list or select **All workspaces** for a full list.
```

File: docs/admin-guide/configuration.md [hunk 2]
```diff
@@ -5,22 +5,23 @@ Configuration is started in Bitbucket and completed in Mattermost.
...
+   * Click **Settings** on the left sidebar to open the Workspace settings.
```
Comment (justinegeffen) [inline docs/admin-guide/configuration.md]:
```suggestion
   * Select **Settings** in the left sidebar to open the workspace settings.
```

File: docs/admin-guide/configuration.md [hunk 3]
```diff
@@ -5,22 +5,23 @@ Configuration is started in Bitbucket and completed in Mattermost.
...
+   * Click **OAuth consumers** under **Apps and features** on the left navigation.
```
Comment (justinegeffen) [inline docs/admin-guide/configuration.md]:
```suggestion
   * Under **Apps and features**, select **OAuth consumers**.
```

**Review by justinegeffen — COMMENTED** [review]
Some editorial feedback. Thank you!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks @sanjaydemansol! Would you be open to updating this branch so it's in sync with the base branch? Thank you!

**Review by DHaussermann — APPROVED** [review]
LGTM!

**Pull request merged**